test/remote/gateways/remote_pay_trace_test.rb

```diff
@@ -17,11 +17,11 @@ def setup
     @gateway = PayTraceGateway.new(fixtures(:pay_trace))
 
     @amount = 100
-    @credit_card = credit_card('4012000098765439')
-    @mastercard = credit_card('5499740000000057')
+    @credit_card = credit_card('4012000098765439', verification_value: '999')
+    @mastercard = credit_card('5499740000000057', verification_value: '998')
     @invalid_card = credit_card('54545454545454', month: '14', year: '1999')
-    @discover = credit_card('6011000993026909')
-    @amex = credit_card('371449635392376')
+    @discover = credit_card('6011000993026909', verification_value: '996')
+    @amex = credit_card('371449635392376', verification_value: '9997')
```

Comment:
all of these cvv updates are due to their testing values requiring specific cvvs based on card brand now.